src/ResourceManager/DataFactories/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 ## Current Release
 
+## Version 3.2.0
```

Comment:
@nicolaszhang please remove the `## Version 3.2.0` header and place the notes that you added under the `## Current Release` header